**ЗАДАЧА**:

По группе семей с доходом 154 руб./чел. зафиксированы следующие цифры потребления молока за месяц (на одного человека):

8,3; 8,6; 8,7; 8,8; 9,1; 9,3; 9,4; 13,4; 13,5; 13,8; 13,9; 14,1; 14,3.

Найти доверительный интервал для **математического ожидания** и **дисперсии** с надежностью 0.95, дать точность оценки.

In [1]:
# импортируем библиотеки
import numpy as np
from bootstrap_CI import bootstrap_samples, confidence_interval

# фиксируем seed 
np.random.seed(42)

# выборка
X = np.array([8.3, 8.6, 8.7, 8.8, 9.1, 9.3, 9.4, 13.4, 13.5, 13.8, 13.9, 14.1, 14.3])

# генерируем новые выборки и считаем статистики

# мат ожидание
stats_mean = [np.mean(sample) for sample in bootstrap_samples(X, 1000)]
# дисперсия
stats_var = [np.var(sample) for sample in bootstrap_samples(X, 1000)]

# создаем доверительные интервалы 

# мат ожидание
mean_interval = confidence_interval(stats_mean, 0.05)
# дисперсия
var_interval = confidence_interval(stats_var, 0.05)

# выводим интервалы
print(f'Доверительный интервал для мат ожидания: {mean_interval}')
print(f'Доверительный интервал для дисперсии: {var_interval}')

Доверительный интервал для мат ожидания: [9.869038461538462, 12.484807692307692]
Доверительный интервал для дисперсии: [3.3625325443786984, 6.9954053254437865]


**ОТВЕТ**:

С вероятностью 95% **математическое ожидание** цифр потребления молока за месяц будет лежать между 9.87 и 12.48.

С вероятностью 95% **дисперсия** цифр потребления молока за месяц будет лежать между 3.36 и 6.99.